shared/teams/add-members-wizard/add-email.tsx

```diff
@@ -4,26 +4,48 @@ import * as Styles from '../../styles'
 import * as Container from '../../util/container'
 import * as TeamsGen from '../../actions/teams-gen'
 import * as Types from '../../constants/types/teams'
+import * as RPCGen from '../../constants/types/rpc-gen'
 import {ModalTitle} from '../common'
 
 type Props = {
   teamID: Types.TeamID
   errorMessage: string
 }
 
+const waitingKey = 'emailLookup'
+
 const AddEmail = (props: Props) => {
   const [invitees, setInvitees] = React.useState('')
+  const [error, setError] = React.useState('')
 
   const dispatch = Container.useDispatch()
   const nav = Container.useSafeNavigation()
   const onBack = () => dispatch(nav.safeNavigateUpPayload())
 
   const disabled = invitees.length < 1
+  const waiting = Container.useAnyWaiting(waitingKey)
 
   const teamID = Container.useSelector(s => s.teams.addMembersWizard.teamID)
 
-  // TODO Y2K-1556 useRPC to get assertions to pass to this action
-  const onContinue = () => dispatch(TeamsGen.createAddMembersWizardPushMembers({members: []}))
+  const emailsToAssertionsRPC = Container.useRPC(RPCGen.userSearchBulkEmailOrPhoneSearchRpcPromise)
+  const onContinue = () => {
+    setError('')
+    emailsToAssertionsRPC(
+      [{emails: invitees}, waitingKey],
+      r =>
+        r?.length
+          ? dispatch(
+              TeamsGen.createAddMembersWizardPushMembers({
+                members: r.map(m => ({
+                  assertion: m.foundUser ? `${m.username}+${m.assertion}` : m.assertion,
```

Comment:
I don't think that's needed because these assertions will be server trust anyway. I faintly remember some functions even throwing it away if you try to do a compound assertion with a servertrust assertion because that doesn't buy any security

Reply:
I think it's mainly for the display on the next page. But maybe we can pass it to that page more structured and then upon adding just add the person.

Reply:
@zapu is the error something like "Unmet remote assertions for user"? Seeing this when I try to hover email assertions.